src/stories/Seekers/CanAddSeeker/endpoint.spec.js

```diff
@@ -76,11 +79,11 @@ describe("Test API Seekers/CanAddSeeker", () => {
     expect(respondResult.json()).toMatchObject({
       uuid: expect.any(String),
       user_uuid: contextClassRef.user.user_uuid,
-      name: "Shubham Joshi",
+      name: "Amarendra Varma",
       seeker_description: {
         gender: "male",
-        age: 25,
-        experience: "4 years",
+        age: 24,
+        experience: "6 months",
```

Comment:
let's store experience in months always?
the use can enter the following values:
1. 1 year => 12 months => 12
2.  7 months => 7
3. 5 years 5 months => 65

Similarly while querying:
If the recruiter asks for applicants having more than 3 years of experience,
we'll filter applicants who are having more than 36 months of experience.

So we can name this key as is to be names as `experience_in_months`